docs/index.md

```diff
@@ -24,6 +26,14 @@ resource "logdna_view" "http500" {
 }
 ```
 
+## Pre-requirements and considerations
+Before using Terraform for creating resources in LogDNA, review the following notes:
+- Verify Terraform is installed. The minimum supported version is 0.12.0.
+- Have the service key for your Organization available. To obtain the service key for your LogDNA Organization, go to the LogDNA dashboard and navigate to Settings > Organization > API Keys.
+- Be aware that the underlying LogDNA Configuration API has a rate limit of 50 requests at a time, therefore when using the LogDNA Terraform provider there is also a limit of 50 resource operations at a time
+- If you do not provide a specific url, the url defaults to "https://api.logdna.com".
+- Authentication is handled via the servicekey parameter.
```

Comment:
Two more tweaks, sorry! ;-)
1. replace second to last sentence with:
If you do not provide a specific base url, the base url defaults to "https://api.logdna.com".
2. replace last sentence above with: 
Authentication is handled via the `servicekey` parameter, which can be passed in the Header of the request (using -H) or, alternatively, in the `username` (-u) (leaving the password empty).

Reply:
for the purposes of using the provider I don't think the user needs to know the second part- I think that is something we should have in our config-api readme.io docs (I think they just need to know that the servicekey is configurable in the provider configuration)

Reply:
Agreed, in the terraform provider the user shouldn't need to be aware of the underlying auth mechanisms the provider takes care of for them. That way they're properly abstracted from the underlying protocols we're using in our provider :D (hey what if one day we switch to graphql or grpc)